Dockerfile.bundle-types

```diff
@@ -0,0 +1,42 @@
+# syntax=docker/dockerfile:1
+
+FROM --platform=linux/amd64 golang:1.19.0 AS protoc
+WORKDIR /protoc
+
+# Install latest protoc binary from https://github.com/protocolbuffers/protobuf/releases
+RUN apt-get update \
+  && apt-get -f install unzip \
+  && rm -rf /var/lib/apt/lists/*
+ARG PROTOC_VERSION=21.5
+ARG PROTOC_SHASUM=92fb4f5066a6f7b870e09c73115a2c861852af8f6555d8da9955fdb80710bf7f
+RUN curl -o /tmp/protoc.zip -L https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-x86_64.zip \
+  && (echo "${PROTOC_SHASUM} /tmp/protoc.zip" | sha256sum -c) \
+  && unzip -o /tmp/protoc.zip -d /protoc bin/protoc \
+  && rm /tmp/protoc.zip
+
+###############
+FROM node:16.17
+
+WORKDIR /client
+
+# Download proto definitions for .sigstore bundle from sigstore/cosign
+ARG REPO=sigstore/cosign
+ARG REF=a8ea54021bb6f850b30ee22bd15e0477d623e00d
```

Comment:
Bleeding edge 😄  looks like this is referencing https://github.com/sigstore/cosign/pull/2204/files